examples/SqlDatabase/SqlDatabase.Tests.ps1

```diff
@@ -1,81 +1,92 @@
 BeforeAll {
   Import-Module Az.InfrastructureTesting
+
+  $Script:rgName = 'rg-test'
+  $Script:location = 'westus3'
+  $Script:serverName = 'samplesqlserver'
+  $Script:databaseName = 'samplesqldatabase'
 }
 
-Describe 'Verify Sql Database Exists' {
-  it 'Should contain a Sql Database with the given name' {
+Describe 'Verify Sql Database' {
+  BeforeAll {
+    $Script:nodatabaseName = 'nosamplesqlserver'
```

Comment:
For these `BeforeAll` vars, could we do the full camel case? Some of them are missing it.

```suggestion
    $Script:noDatabaseName = 'nosamplesqlserver'
```